playbooks/tasks/install-app.yml

```diff
@@ -1 +1,26 @@
 ---
+
+- set_fact:
+    siteworx_docroot: /home/{{ siteworx_user }}/{{ siteworx_domain }}/html
+
+- name: Check if Magento2 already installed
```

Comment:
The install process from here needs to be rethought.  I will be passing a type indicator to let you know what app will be being installed from the following list:

https://github.com/clwells/core/blob/97edf29a49396faf3e4ee36f75264ccc0ca6be5e/nocworx/db/Migrations/20180225171259_nsd4857.php#L41-L47

I will also pass version info ('1' or '2' for example for magento).  Finally, I'll be passing admin user/pass info too and anything else that's needed by default (will need to know what this is).